cmd/juju/cloud/show.go

```diff
@@ -95,9 +100,16 @@ type cloudDetails struct {
 	// Regions is for when we want to print regions in order for yaml output.
 	Regions yaml.MapSlice `yaml:"regions,omitempty" json:"-"`
 	// Regions map is for json marshalling where format is important but not order.
-	RegionsMap   map[string]regionDetails `yaml:"-" json:"regions,omitempty"`
-	Config       map[string]interface{}   `yaml:"config,omitempty" json:"config,omitempty"`
-	RegionConfig jujucloud.RegionConfig   `yaml:"region-config,omitempty" json:"region-config,omitempty"`
+	RegionsMap      map[string]regionDetails `yaml:"-" json:"regions,omitempty"`
+	BootstrapConfig map[string]interface{}   `yaml:"bootstrap-config,omitempty" json:"bootstrap-config,omitempty"`
+	ModelConfig     map[string]interface{}   `yaml:"model-config,omitempty" json:"model-config,omitempty"`
+	RegionConfig    jujucloud.RegionConfig   `yaml:"region-config,omitempty" json:"region-config,omitempty"`
+	ProviderConfig  map[string]interface{}   `yaml:"supported-cloud-config,omitempty" json:"supported-cloud-config,omitempty"`
```

Comment:
Or just "cloud-config".
That then fits with "model-config", "region-config" etc.